File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int a[] = new int[2];
        int index= 0;
        int num=0;
        for(int i=0;i<nums.length;i++){
            index=i;
            num = nums[i];
            for(int j=i+1;j<nums.length;j++){
                if(num+nums[j] == target){
                    a[0]=i;
                    a[1]=j;
                    return a;
                }
            }
        }
        return a;
    }
}

File: 0001-two-sum/0001-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    for(x in nums){
        let sum = nums[x];
        for(y in nums){
            if(x !== y && nums[x] + nums[y] === target){
                return [x, y];
            }
        }
    }
};